packages/data/src/plugins/persistence/storage/object.js

```diff
@@ -0,0 +1,23 @@
+let _objectStorage;
```

Comment:
Do we have usage of `_` as a prefix documented? I did a quick search and found only one place where we use `let _variable`. With ES6 modules it doesn't seem to be so important to use this patter to indicate that variable is private.

Reply:
I agree. I'll remove the `_` prefix.